src/reporter/moment-duration-format-loader.js

```diff
@@ -0,0 +1,33 @@
+const resolveFrom              = require('resolve-from');
+const testcafeMomentModulePath = require.resolve('moment');
```

Comment:
I think, we also must be carefull with TestCafe-related module. Let's handle it with the following algorithm:
```
const originalModule = cache[modulePath];

delete cache[modulePath];

const module = require(modulePath);
// ...
// do stuff
// ...

if (originalModule)
    cache[modulePath] = originalModule;
else
    delete cache[originalModule];
```